libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/ExecSpecRule/Conway.hs

```diff
@@ -391,38 +394,29 @@ instance
   signalSpec _ env st =
     txCertSpec env st
       <> constrained disableRegCerts
-      <> constrained disableDRepRegCerts
+      <> constrained disableDRepRegCerts'
     where
       disableRegCerts :: Term fn (ConwayTxCert Conway) -> Pred fn
       disableRegCerts cert =
-        (caseOn cert)
+        caseOn
+          cert
```

Comment:
```suggestion
        (caseOn cert)
```
Let's keep the parentheses like they were, makes it look nicer in this case I think

Reply:
Oh yes, it looks bad after `fourmolu` ugly-fied it :grin: